engine/components-api/src/main/java/com/cloud/configuration/ConfigurationManager.java

```diff
@@ -54,10 +55,40 @@
  */
 public interface ConfigurationManager {
 
-    public static final String MESSAGE_CREATE_POD_IP_RANGE_EVENT = "Message.CreatePodIpRange.Event";
-    public static final String MESSAGE_DELETE_POD_IP_RANGE_EVENT = "Message.DeletePodIpRange.Event";
-    public static final String MESSAGE_CREATE_VLAN_IP_RANGE_EVENT = "Message.CreateVlanIpRange.Event";
-    public static final String MESSAGE_DELETE_VLAN_IP_RANGE_EVENT = "Message.DeleteVlanIpRange.Event";
+    String MESSAGE_CREATE_POD_IP_RANGE_EVENT = "Message.CreatePodIpRange.Event";
+    String MESSAGE_DELETE_POD_IP_RANGE_EVENT = "Message.DeletePodIpRange.Event";
+    String MESSAGE_CREATE_VLAN_IP_RANGE_EVENT = "Message.CreateVlanIpRange.Event";
+    String MESSAGE_DELETE_VLAN_IP_RANGE_EVENT = "Message.DeleteVlanIpRange.Event";
+
+    ConfigKey<Boolean> SystemVMUseLocalStorage = new ConfigKey<>(
+            Boolean.class,
+            "system.vm.use.local.storage",
+            "Advanced",
+            "false",
+            "Indicates whether to use local storage pools or shared storage pools for system VMs.",
+            false,
+            ConfigKey.Scope.Zone,
+            null);
+
+    ConfigKey<String> ControlCidr = new ConfigKey<>(
+            String.class,
+            "control.cidr",
+            "Advanced",
+            "169.254.0.0/16",
```

Comment:
```suggestion
            "169.254.240.0/20",
```
like this and abandon all other changes in this PR, right @wido ?

Reply:
Almost @DaanHoogland , but also in:

- NetUtils.java
- AgentProperties.java
- configuration/Config.java

If we change it to a /20 in those files we should be good.

Reply:
This will only work for new environments. I don't think this will work when upgrading environment. These hardcoded values of an already otherwise deplayed config item will break systems and needs a bit more TLC .

Reply:
@DaanHoogland 
as I understand, Wido wants /20 to be the default in new environments.
It would be good to update the config description to emphasize that the new value only applies on new pods.

Reply:
well, this means that the hardcoded values in NetUtils.java and AgentProperties.java have to be intelligent... we can not assume returning the one or the other in these methods. That complicates things a bit.